spec/services/claims/clawback/generate_csv_file_spec.rb

```diff
@@ -21,9 +21,10 @@
            group: "Independent schools",
            type_of_establishment: "Other independent school")
   end
-  let(:claim_1) { create(:claim, :submitted, status: :clawback_in_progress, school: school_a, reference: "11111111") }
-  let(:claim_2) { create(:claim, :submitted, status: :clawback_in_progress, school: school_b, reference: "22222222") }
-  let(:claim_3) { create(:claim, :submitted, status: :clawback_in_progress, school: school_b, reference: "33333333") }
+  let(:provider) { create(:claims_provider, name: "Springfield Trust") }
+  let(:claim_1) { create(:claim, :submitted, status: :clawback_in_progress, school: school_a, reference: "11111111", provider: provider) }
+  let(:claim_2) { create(:claim, :submitted, status: :clawback_in_progress, school: school_b, reference: "22222222", provider: provider) }
+  let(:claim_3) { create(:claim, :submitted, status: :clawback_in_progress, school: school_b, reference: "33333333", provider: provider) }
```

Comment:
```suggestion
  let(:claim_1) { create(:claim, :submitted, status: :clawback_in_progress, school: school_a, reference: "11111111", provider:) }
  let(:claim_2) { create(:claim, :submitted, status: :clawback_in_progress, school: school_b, reference: "22222222", provider:) }
  let(:claim_3) { create(:claim, :submitted, status: :clawback_in_progress, school: school_b, reference: "33333333", provider:) }
```

Can we omit `provider` here?